plugins/dstat_nvidia_gpu.py

```diff
@@ -0,0 +1,41 @@
+### Author: Vasilis Vryniotis <bbriniotis@datumbox.com>
+
+class dstat_plugin(dstat):
+    """
+    Total GPU usage for NVIDIA cards. Requires the nvidia-ml-py package.
+
+    Usage: dstat --nvidia-gpu
+    """
+
+    def __init__(self):
+        self.name = 'gpu nvidia'
+        self.nick = ('gpu',)
+        self.vars = ('gpu',)
+        self.type = 'p'
+        self.width = 5
+        self.scale = 1
+
+    def check(self):
+        import pynvml
```

Comment:
Hehe... My thoughts exactly. Your code should be able to handle ion the Dstat class. Originally I planned to try-catch it but then I saw that Python 2 and 3 have different way to check if the module exists. Is your code compatible with both python versions?